package.json

```diff
@@ -36,5 +37,8 @@
     "scripts": {
         "test": "nyc mocha tests/rules/",
         "lint": "eslint ."
+    },
+    "dependencies": {
+        "typescript": "^3.9.5"
```

Comment:
Do we need that as a dependency? :thinking:

Reply:
oops - moved to dev dep